src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -1,9 +1,257 @@
+    if (this.state.src) instruction = 'Step 2 of 2: Fit your head and shoulders in the frame';
+    if (this.state.done) description = 'Success! This photo will be printed on your Veteran ID card.';
+
     return (
-      <div>Imagine a photo upload widget here</div>
+      <div style={{ border: 'lightgrey 9px solid' }}>
+        <div style={{ padding: '2px' }}>
+          <h3>Photo upload <span className="form-required-span">(Required)*</span></h3>
+          {!this.state.done && <p>{instruction}</p>}
+          {(this.state.src || this.state.done) && <p>{description}</p>}
+          {this.state.done && <img style={{ width: '100%' }} src={this.state.cropResult} alt="cropped"/>}
+        </div>
+        {this.state.src && <div>
+          <Cropper
+            ref="cropper"
+            responsive={false}
+            src={this.state.src}
+            aspectRatio={1 / 1}
+            cropBoxMovable={false}
+            toggleDragModeOnDblclick={false}
+            dragMode="move"
+            guides={false}
+            viewMode={1}
+            zoom={this.onZoom}
+            crop={this.cropImage}/>
+          <input type="range"
+            ref="slider"
+            min="2"
+            max="10"
+            defaultValue="1"
+            step="0.1"
+            aria-valuemin="2"
+            aria-valuemax="10"
+            aria-valuenow={this.state.zoomValue}
+            onInput={this.zoom}/>
+          <div className="cropper-control-column">
+            <button className="cropper-control va-button va-button-link" type="button" onClick={this.zoomOut}>
+              Make smaller
+            </button>
+            <button className="cropper-control va-button-link" type="button" onClick={this.zoomIn}>
+              Make larger
+            </button>
+          </div>
+          <div className="cropper-control-contrainer">
+            <div className="cropper-control-column">
+              <button className="cropper-control va-button-link" type="button" onClick={this.moveUp}>
+                Move up
+              </button>
+              <button className="cropper-control va-button-link" type="button" onClick={this.moveDown}>
+                Move down
+              </button>
+            </div>
+            <div className="cropper-control-column">
+              <button className="cropper-control va-button-link" type="button" onClick={this.moveRight}>
+                Move right
+              </button>
+              <button className="cropper-control va-button-link" type="button" onClick={this.moveLeft}>
+                Move left
+              </button>
+            </div>
+          </div>
+          <button type="button" onClick={this.onDone} style={{ 'float': 'right' }}>
+            I'm done
+          </button>
+        </div>
+        }
+        {!this.state.src && !this.state.done && <Dropzone onDrop={this.onChange} accept="image/jpeg, image/png"/>}
```

Comment:
This accept list is different from the allowed file types in other places.

Reply:
Good catch, updated.